add-ons/dspace-docker-tools/makeAIP.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+ADMIN_EMAIL=${ADMIN_EMAIL:-test@test.edu}
+AIPOUT=/tmp/aipout
+rm -rf ${AIPOUT}
```

Comment:
Same as above: please add `echo` and `rm -i`.

Reply:
see above